Gaijinizer patch/CommonEvents_428.txt

```diff
@@ -4,80 +4,96 @@
 
 絶対に離さなイヨ…♡
 > CONTEXT: CommonEvents_428/14/Dialogue/enemy13
-No...♡. Iyo absolutely Iyo...♡#MTLed
+nO...♡
+
+I wiLL abSOluteLY not leT yOU gO...♡#Strate
 > END STRING
 
 > BEGIN STRING
 いやよ…！！
 
 絶対に離さないから…！
 > CONTEXT: CommonEvents_428/20/Dialogue/enemy12
-no…! ! From I won't let go ...!#MTLed
+No...!!
+
+I absolutely won't let you go...!#Strate
 > END STRING
 
 > BEGIN STRING
 ………離シて欲しイ……の？
 > CONTEXT: CommonEvents_428/30/Dialogue/enemy13
-......... I want to leave me .........#MTLed
+.........yoU want mE to......leT yoU Go?#Strate
 > END STRING
 
 > BEGIN STRING
 離して欲しいの……？
 
 …………
 > CONTEXT: CommonEvents_428/34/Dialogue/enemy12
-Do you want me to leave ...? ............#MTLed
+You want me to let you go......?
+
+............#Strate
 > END STRING
 
 > BEGIN STRING
 …………今回だケ
 
 特別……ダよ…♡
 > CONTEXT: CommonEvents_428/44/Dialogue/enemy13
-............ This time. Special ...... Da......
-♡#MTLed
+............onLY thiS OncE.
+
+I'll mAke......aN exCepTion......♡#Strate
 > END STRING
```

Comment:
6 ellipses on the first line, and shorten each section of ellipses in the second line to 3 each.